lib/blockstore/index.js

```diff
@@ -0,0 +1,42 @@
+/*!
+ * blockstore/index.js - bitcoin blockstore for bcoin
+ * Copyright (c) 2019, Braydon Fuller (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+const {join} = require('path');
+
+const AbstractBlockStore = require('./abstract');
+const LevelBlockStore = require('./level');
+const FileBlockStore = require('./file');
+
+/**
+ * @module blockstore
+ */
+
+exports.create = (options) => {
```

Comment:
Would a user want to ever run the old leveldb database but still use new versions of bcoin? It seems like this is a forced migration, which it is an upgrade so it doesn't seem like that big of a deal

Reply:
I don't think so, there is a migration script for moving the block data and it doesn't take long.